### DISCUSSION TITLE
Duplicate Entry Issue (fixing in docker-compose / portainer)

### DISCUSSION THREAD

**Discussion by DSew20** [photoprism/photoprism#2453] [Q&A]
Hello all,

After fixing some earlier issues related to PATHS and issues for Ubuntu 21.04, I have hit another issue regarding "duplicate entry".

Does anybody know how to fix the below problem whilst running in under Portainer?

```
running as uid 0
photoprism start
time="2022-06-15T20:55:13Z" level=debug msg="entity: migrate Error 1062: Duplicate entry 'Innsbruck, Tirol, Austria' for key 'uix_places_place_label' (waiting 1s)"
panic: Error 1062: Duplicate entry 'Innsbruck, Tirol, Austria' for key 'uix_places_place_label'

```

I really want to get Photoprism running because I have invested lots of time setting up, face recognition, tagging etc.  But, I can not start the container I am concerned I will loose all of my config.

This discussion is related to an earlier issue #2426
I have also tried to find a solution through the Portainer page https://github.com/portainer/portainer/issues/7087

Kind regards

**Comment by lastzero** [comment]
Seems migrations didn't run as the unique index was replaced with  a regular index, so it should not happen. I don't have time to search right now, but you should find related reports on GitHub. Make sure to also check closed issues and discussions.


To see the status of migrations, run this in a terminal:

```
photoprism migrations ls
```

If any migrations have failed, re-run those using this command:

```
photoprism migrations run -f
```
